dm/dm/config/checking_item.go

```diff
@@ -52,6 +53,7 @@ var AllCheckingItems = map[string]string{
 	ShardAutoIncrementIDChecking: "conflict auto increment ID of shard tables checking item",
 	OnlineDDLChecking:            "online ddl checking item",
 	BinlogDBChecking:             "binlog db checking item",
+	ConnAmountChecking:           "conn amount checking item",
```

Comment:
```suggestion
	ConnAmountChecking:           "connection amount checking item",
```

this is the displayed message, should be readable